label_studio/users/migrations/0007_user_is_deleted.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.20 on 2023-09-18 22:18
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('users', '0006_user_allow_newsletters'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='user',
+            name='is_deleted',
+            field=models.BooleanField(default=False, help_text='Designates whether to treat this user as deleted. Select this instead of deleting accounts.', verbose_name='deleted'),
```

Comment:
Is this supposed to say "Select" or "Unselect" like the field?

Reply:
I meant select as I wrote it as selecting a boolean field would suggest marking it true for deleted.  But maybe select is not the best word here

Reply:
Ahh yes the field is wrong then, they should both say select